### PR TITLE
added dither

### PR DESCRIPTION
**Pull request by 1Euro7Cent**
I also fixed that test module being in production dependencies. Modules like that are usually in devDependencies

You need to update that Firefox logo in the README if you want.

And sorry again for auto formatting

### CODE REVIEW

**Review by charleprr — CHANGES_REQUESTED** [review]
Hello,

I've added a few suggestions before merging your pull request.
Could you add `package-lock.json` back too?

Also, the code formatting actually looks a bit better now.

Thanks for your help! 🙂

File: index.js [hunk 1]
```diff
@@ -1,4 +1,6 @@
+// @ts-nocheck
```
[the remark this answers is not in the document]
Comment (1Euro7Cent) [reply index.js:1]: I use typescript checking and i cannot resolve a error on that line (it works how it is). That disables checking for that line below

**Comment by 1Euro7Cent** [comment]
Im gonna resolve the rest when i am on my pc

File: index.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (1Euro7Cent) [reply index.js:1]: Ughh ts nocheck disables the checking for the whole file

[the remark this answers is not in the document]
Comment (1Euro7Cent) [reply index.js:1]: if i remove the tsnocheck this happens: 
image
  but ill remove it

**Comment by 1Euro7Cent** [comment]
I don't know if you noticed it but I resolved it

**Review by charleprr — APPROVED** [review]

**Comment by charlypoirier** [comment]
One last issue, I get this error when trying to launch Emojicon.

```error
node:internal/modules/cjs/loader:936
  throw err;
  ^

Error: Cannot find module './baseDither'
Require stack:
- /.../projects/emojicon-dev/node_modules/multidither/classes/dither/FloydSteinbergDither.js
- /.../projects/emojicon-dev/node_modules/multidither/index.js
- /.../projects/emojicon-dev/index.js
- ...
```

I think it is because it is trying to import `baseDither.js` but the real file is named `BaseDither.js`.
Do you think you could fix that?

**Comment by 1Euro7Cent** [comment]
for me it works now

**Comment by charlypoirier** [comment]
Thanks!

**Pull request merged**